optimade/server/config.py

```diff
@@ -1,260 +1,68 @@
 import os
 import json
-from typing import Any
+from typing import Any, Optional, Dict, List
+from typing_extensions import Literal
 from configparser import ConfigParser
 from pathlib import Path
 from warnings import warn
 
+from pydantic import BaseSettings, root_validator
```

Comment:
I don't see you using this, but maybe that's my fault for unrequested reviewing a WIP PR ...
```suggestion
from pydantic import BaseSettings
```

Reply:
You're correct.